request_a_govuk_domain/request/templates/base.html

```diff
@@ -66,18 +66,20 @@
 
   {% block phase_banner %}
     <div class="govuk-phase-banner govuk-width-container">
-      <p class="govuk-phase-banner__content">
+      <div class="govuk-phase-banner__content">
```

Comment:
Why this change?

Reply:
> Why this change?

Wanted to keep HEADER and CONTENT of banner in one line.